cmd/gateway-main/main.go

```diff
@@ -0,0 +1,13 @@
+package main
+
+import "time"
+
+func IsGatewayV2() bool {
```

Comment:
we can remove this function since we are running gateway logic in a different container.

Reply:
removed